microsoft-edge/web-platform/user-agent-guidance.md

```diff
@@ -159,19 +163,31 @@ For legacy reference, the following format was used for User-Agent string.
 On Windows, the `User-Agent` HTTP request header uses the following format:
 
 ```https
-User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36 Edg/90.0.818.46
+User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64)  
+    AppleWebKit/537.36 (KHTML, like Gecko)  
+    Chrome/90.0.4430.85  
+    Safari/537.36  
+    Edg/90.0.818.46
 ```
 
 On Android, the `User-Agent` HTTP request header uses the following format:
 
 ```https
-User-Agent: Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Mobile Safari/537.36 EdgA/90.0.818.46
+User-Agent: Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N)  
+    AppleWebKit/537.36 (KHTML, like Gecko)  
+    Chrome/90.0.4430.85  
+    Mobile Safari/537.36  
+    EdgA/90.0.818.46
 ```
 
 The response value from `navigator.userAgent` method uses the following format:
 
 ```javascript
-"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4501.0 Safari/537.36 Edg/91.0.866.0"
+"Mozilla/5.0 (Windows NT 10.0; Win64; x64)  
+AppleWebKit/537.36 (KHTML, like Gecko)  
+Chrome/91.0.4501.0  
+Safari/537.36  
+Edg/91.0.866.0"

```

Comment:
Now, this one is different, because it's the return value from the `navigator.userAgent` property. It's a JavaScript string. So I don't think we should wrap it on multiple lines. In fact it's within `"` characters which, in JavaScript, don't support line wraps. So let's leave this one unchanged.

Reply:
Okay